src/sparseml/pytorch/models/classification/efficientnet.py

```diff
@@ -82,26 +82,12 @@ def forward(self, a: Tensor, b: Tensor):
             return torch.add(a, b)
 
 
-class QATSiLU(SiLU):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.wrap_qat = True
-        self.qat_wrapper_kwargs = {
-            "num_inputs": 1,
-            "num_outputs": 0,
-        }
-
-
-class QATSiLUOutput(SiLU):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class MulInput(Module):
```

Comment:
PyTorch has an `Identity` Module as part of its `nn` library. Why don't we just use that?

Reply:
The identity module is not quantized by default